swap_meet/clothing.py

```diff
@@ -1,2 +1,9 @@
-class Clothing:
-    pass
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, condition = 0):
+        self.category = "Clothing"
+        self.condition = condition
```

Comment:
While the parent passed as an argument in the class definition, the real benefits of inheritance are not being utilized here. The parent constructor allows the child class to add values to the inherited attributes from the parent. Instead what's happening here is that `Clothing` is re-creating category/condition attributes and not inheriting category/condition from the parent. We can reduce this entire constructor code to 2 lines by using inheritance:

```suggestion
class Clothing(Item):
    def __init__(self, condition = 0, age = 0):
        super().__init__(category="Clothing, condition = condition, age = age)
```